RickAndMorty-iOS/Coordinators/MainCoordinator.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  MainCoordinator.swift
+//  RickAndMorty-iOS
+//
+//  Created by Calvin Pak on 2023-03-08.
+//
+
+import UIKit
+
+class MainCoordinator: Coordinator {
+
+    let window: UIWindow
+
+    var tabBarController = UITabBarController()
+
+    let characterNavController = UINavigationController()
+    let locationNavController = UINavigationController()
+    let episodeNavController = UINavigationController()
+    let searchNavController = UINavigationController()
+
+    func customNavBarAppearance() -> UINavigationBarAppearance {
```

Comment:
wee need to follow certain order when we declare variables and functions. 
basical idea is 
protocol properties. 
class properties
lifecycle methods
other methods

https://neofinancial.getoutline.com/doc/property-declaration-order-zNGNgeEgzx

Reply:
Hi @neo-valerii-iamkovyi,  access to the above URL requires login, could you please copy the contents from the doc to us for reference? (or share it to our Slack channel so that we can keep the contents private) 

Thanks.